### ISSUE TITLE
2.2.6 does not install on macOS Catalina

### ISSUE THREAD

**Issue by nlfiedler** [hashicorp/vagrant#11127]
Had 2.2.5 installed earlier, prior to upgrading to macOS Catalina. Then I updated to Catalina. Now I tried to install 2.2.6 (after removing 2.2.5) and the Installer.app tells me that the "software needs to be updated" because it cannot tell if it is malicious or not. Probably needs to be signed using the new process Apple demands.

Meanwhile I have no vagrant. Advice would be appreciated. Thanks.

**Comment by nlfiedler** [comment]
Found a workaround:
```shell
$ sudo installer -allowUntrusted -pkg /Volumes/Vagrant/vagrant.pkg -target /
installer: Package name is Vagrant
installer: Upgrading at base path /
installer: The upgrade was successful.

$ which vagrant
/usr/local/bin/vagrant

$ vagrant version
Installed Version: 2.2.6
Latest Version: 2.2.5

You're running an up-to-date version of Vagrant!
```

**Comment by chrisroberts** [comment]
Hi there. 

There are new restrictions for applications imposed by Apple starting with Catalina. Included in these new restrictions is the requirement for notarization which requires submission of the application build to Apple and an acceptance before it can be packaged and distributed.

We are working on integrating this process into our build system and will have it available for our next release. Until then, disabling the check (as you have done above) will allow for installation.

Cheers!

**Comment by muratgozel** [comment]
You also may just allow it from `System Preferences - Security & Privacy - Allow apps downloaded from` section without coding. 

<img width="665" alt="Screen Shot 2019-10-20 at 21 38 19" src="https://user-images.githubusercontent.com/15162371/67164340-4296fc00-f382-11e9-9c84-d44d5f96cd55.png">

**Comment by timcoote** [comment]
> You also may just allow it from `System Preferences - Security & Privacy - Allow apps downloaded from` section without coding.
> 
> <img alt="Screen Shot 2019-10-20 at 21 38 19" width="665" src="https://user-images.githubusercontent.com/15162371/67164340-4296fc00-f382-11e9-9c84-d44d5f96cd55.png">

This approach did not work for me: I could not identify the relevant s/w author from the list of blocked authors, nor add another. (after clicking on the padlock at the bottom left and opening the 'Allow apps downloaded from:' option.

**Comment by patrickgilmour** [comment]
> > You also may just allow it from `System Preferences - Security & Privacy - Allow apps downloaded from` section without coding.
> > <img alt="Screen Shot 2019-10-20 at 21 38 19" width="665" src="https://user-images.githubusercontent.com/15162371/67164340-4296fc00-f382-11e9-9c84-d44d5f96cd55.png">
> 
> This approach did not work for me: I could not identify the relevant s/w author from the list of blocked authors, nor add another. (after clicking on the padlock at the bottom left and opening the 'Allow apps downloaded from:' option.

It worked here.

- Try running the installer.
- When it says is can't be installed, open the security prefs
- You should now see an "open anyway" button. Click it.
- Return to the installer and there is now a second button that allows you to install.

**Comment by chrisroberts** [comment]
This has been resolved in the latest release (2.2.7).
